EventFilter/CastorRawToDigi/python/CastorRawToDigi_cfi.py

```diff
@@ -28,5 +29,13 @@
     InputLabel = cms.InputTag("rawDataCollector"),
     CastorCtdc = cms.bool(False),
     UseNominalOrbitMessageTime = cms.bool(True),
-    ExpectedOrbitMessageTime = cms.int32(-1)
-)
+    ExpectedOrbitMessageTime = cms.int32(-1),
+    UnpackZDC = cms.bool(True),
+    #myMap = cms.VPSet(
+     #  cms.PSet(
+      ##     object = cms.string('ElectronicsMap'),
+        #   file = cms.FileInPath('/afs/cern.ch/user/j/jgomez2/ZDCTest/QuickFix/CMSSW_7_5_0/src/EventFilter/CastorRawToDigi/data/old_ZDC_emap.txt')
```

Comment:
please cleanup the "myMap" commented out piece.
at least rename the private directory file name